packages/config/src/layer2s/types/Layer2.ts

```diff
@@ -95,5 +95,5 @@ export interface Layer2TVLWarning {
   /** Content of the warning */
   content: string
   /** Color with which the warning should be displayed */
-  severity: "yellow" | "red"
+  sentiment: 'bad' | 'warning'
```

Comment:
lets pick it from Sentiment type: `Pick<Sentiment, 'bad' | 'warning'>`

Reply:
You can't pick from a union type only from an interface/class

Reply:
Extract was the correct utility type, sorry 👍